source/future.tex

```diff
@@ -134,6 +135,71 @@
 the namespace \tcode{std}.
 \end{example}
 
+\rSec2[ccomplex.syn]{Header \tcode{<ccomplex>} synopsis}
+\indexlibrary{\idxhdr{ccomplex}}%
+
+\begin{codeblock}
+#include <complex>
+\end{codeblock}
+
+\pnum
+The header \tcode{<ccomplex>}
+behaves as if it simply includes the header
+\tcode{<complex>}~(\ref{complex.syn}).
+
+\rSec2[cstdalign.syn]{Header \tcode{<cstdalign>} synopsis}
+
+\indexlibrary{\idxcode{__alignas_is_defined}}%
+\indexlibrary{\idxhdr{cstdalign}}%
+\begin{codeblock}
+#define @\xname{alignas_is_defined}@ 1
+\end{codeblock}
+
+\pnum
+\indexlibrary{\idxhdr{cstdalign}}%
+\indexlibrary{\idxhdr{stdalign.h}}%
+The contents of the header \tcode{<cstdalign>} are the same as the C
+standard library header \tcode{<stdalign.h>}, with the following changes:
+The header \tcode{<cstdalign>} and the header \tcode{<stdalign.h>} shall not
```

Comment:
My biggest concern is providing the specification for the C header in the deprecated C++ header section, and similarly for stdbool.h.  It is also deprecated to use any of the C headers by name in C++, but it still makes me more nervous about this change.